components/style/properties/longhand/margin.mako.rs

```diff
@@ -7,7 +7,12 @@
 <% data.new_style_struct("Margin", inherited=False) %>
 
 % for side in ALL_SIDES:
+    <%
+        spec = "https://drafts.csswg.org/css-box/#propdef-%s" % side[0]
```

Comment:
is there a `margin-` missing here? This will generate links like `propdef-right`, right?